docs/features/pootle_fs_add_project.rst

```diff
@@ -39,7 +39,9 @@ is outlined here:
   - Set the **Filesystem backend** to ``localfs`` as we are working against
     files stored on the local filesystem.
   - Set the **Path or URL** to point to the translation files on
-    Pootle's local filesystem, e.g. ``/path/to/translations/MYPROJECT/``
+    Pootle's local filesystem, e.g. ``{POOTLE_TRANSLATION_DIRECTORY}MYPROJECT``
+    (``{POOTLE_TRANSLATION_DIRECTORY}`` will be transparently replaced by the
+    value of :setting:`POOTLE_TRANSLATION_DIRECTORY` setting)
```

Comment:
I would add the `{POOTLE_TRANSLATION_DIRECTORY}` as a second example.  The way its written now someone will assume that they must use `{POOTLE_TRANSLATION_DIRECTORY}` but it is optional